test/extended/builds/remove_buildconfig.go

```diff
@@ -55,12 +55,19 @@ var _ = g.Describe("[Feature:Builds][Conformance] remove all builds when build c
 
 				g.By("waiting for builds to clear")
 				err = wait.Poll(3*time.Second, 3*time.Minute, func() (bool, error) {
+					buildsDeleted := false
 					out, err := oc.Run("get").Args("builds").Output()
 					o.Expect(err).NotTo(o.HaveOccurred())
 					if out == "No resources found." {
-						return true, nil
+						buildsDeleted = true
 					}
-					return false, nil
+					configMapsDeleted := false
+					out, err = oc.Run("get").Args("configmaps").Output()
+					o.Expect(err).NotTo(o.HaveOccurred())
+					if out == "No resources found." {
```

Comment:
of course this is fragile wrt if the message changes

there are plenty of examples in the test/extended/util/framework.go file where `list` is employed.  For example, there are plenty of listings of pods that look like 

`podList, err := oc.AdminKubeClient().CoreV1().Pods(namespace).List(metav1.ListOptions{})`

IMO I think it would be better to list config maps and work off of an empty list that is returned.

Reply:
@gabemontero updated to list builds and configMaps